src/bld/package.js

```diff
@@ -34,13 +34,13 @@ const packager = async (
   log.debug(`Copy ${nwDir} files to ${outDir} directory`);
   await cp(nwDir, outDir, { recursive: true });
 
-  for (const file of files) {
+  for (let file of files) {
     log.debug(`Copy ${file} file to ${outDir} directory`);
     await cp(
       file,
       `${outDir}/${
         platform !== "osx" ? "package.nw" : "nwjs.app/Contents/Resources/app.nw"
-      }/${basename(file)}`,
+      }/${file.split(sep).splice(2).join(sep)}`,
```

Comment:
This line cause problem on windows. "/" instead of `sep` should be used here (The rest of the path is already using "/". `file` come from glob expanded `srcDir`, which always use "/". `split(sep)` split nothing on windows causing the subdirectory disappear.

Reply:
Can you open a bug report referencing this line with a minimal repro?

Reply:
I'm working on getting Windows builds working in #795. I realise I need to use `path.resolve` which would remove the need for `sep`.